hat/assets/js/apps/Iaso/domains/orgUnits/reviewChanges/index.tsx

```diff
@@ -95,12 +98,26 @@ export const ReviewOrgUnitChanges: FunctionComponent = () => {
     );
 
     const csv_url = getTableUrl(endPointUrl, csv_params);
+    const defaultSourceVersion = useDefaultSourceVersion();
+    const [selectedVersionId, setSelectedVersionId] = useState<string>(
+        defaultSourceVersion.version.id.toString(),
+    );
+    const [dataSource, setDataSource] = useState<string>(
+        defaultSourceVersion.source.id.toString(),
+    );
 
+    params.source_version_id = selectedVersionId;
```

Comment:
I think you could move all this code to the Filters, since you only pass it as props. You could use a redirection from within the filters instead of mutating the `params` here